tools/CleanUp_tool_powershell_script/CleanUp_tool.ps1

```diff
@@ -0,0 +1,31 @@
+﻿$SettingsPath = $Env:LOCALAPPDATA + '\Microsoft\PowerToys'
+
+if (Test-Path -Path $SettingsPath -PathType Any)
+{
+    Remove-Item –Path $SettingsPath –Recurse
+}
+
+$SuperFancyZones = "HKCU:\Software\SuperFancyZones"
+$PowerRename = "HKCU:\Software\Microsoft\PowerRename"
+$ImageResizer = "HKCU:\Software\Microsoft\ImageResizer"
+$DontShowThisDialogAgain = "HKCU:\Software\Microsoft\Windows\CurrentVersion\Explorer\DontShowMeThisDialogAgain"
+
+if (Test-Path -Path $SuperFancyZones -PathType Any)
+{
+    Remove-Item –Path $SuperFancyZones –Recurse
+}
+
+if (Test-Path -Path $PowerRename -PathType Any)
+{
+    Remove-Item –Path $PowerRename –Recurse
+}
+
+if (Test-Path -Path $ImageResizer -PathType Any)
+{
+    Remove-Item –Path $ImageResizer –Recurse
+}
+
+if (Test-Path -Path $DontShowThisDialogAgain -PathType Any)
+{
+    Remove-Item –Path $DontShowThisDialogAgain –Recurse
```

Comment:
Bad idea to remove this recursive because they key is used from other applications and dialogues too.
Please remove only the PT values here!

p. s.: I think the same change would be necessary for the tool's code.

Reply:
Good point.